_docs/collection-details.md

```diff
@@ -145,6 +145,72 @@ Finally once the payment goes through the customer will see a success page:
 
 ![Paga transaction page 4]({{ "/img/collections/paga_mobile_page_4.png" | prepend: site.baseurl }})
 
+# NGN card or bank collection requests through Providus (Beta)
+
+To initiate an NGN bank or card collections through Providus, use the following details:
+
+{% capture data-raw %}
+```javascript
+"input_currency": "NGN",
+"payin_methods": [
+  {
+    "type": "NGN::Bank",
+    "in_details": {
+      "payment_method": "bank", // or "card"
+      "redirect_url": "http://redirect.back.to"
+    },
+    "provider": "providus"
+  }
+],
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="collection-ngn-card-bank" raw=data-raw %}
+
+Please note:
+- The `redirect_url` which has to be set to an URL where you wish the user to be redirected once the payment has either been done, or the request has been rejected by the user.
+- The `payment_method` options are `bank` or `card`. You can leave it empty to support both.
+
+
+Once the transaction has been created the `out_details` will be:
+
+{% capture data-raw %}
+```javascript
+"out_details": {
+  "Bank Name": "Providus Bank",
+  "Account Number": "yyyyyyyy",
+  "url": "https://sdk.monnify.com/checkout/xxxxxxxx"
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="collection-ngn-card-bank-out" raw=data-raw %}
+
+In this case the user needs to be redirected to the url described at `url`.
```

Comment:
Add notes that in case of bank payment they don't actually need to get redirected, as the details they need to do are in the out_details hash above